lib/rules/style-prop-object.js

```diff
@@ -71,6 +95,21 @@ module.exports = {
           return;
         }
 
+        // parent element is a JSXOpeningElement
+        if (node.parent && node.parent.type === 'JSXOpeningElement') {
+          // store parent element
+          const jsxOpeningElement = node.parent;
```

Comment:
this can be stored prior to the "if", so we don't have to extract it multiple times.